examples/ensemble/plot_gradient_boosting_categorical.py

```diff
@@ -69,7 +69,6 @@
 print(f"Number of categorical features: {n_categorical_features}")
 print(f"Number of numerical features: {n_numerical_features}")
 
-# %%
```

Comment:
You need to reintroduce all these markers. They are used to define the sections.
Now, the rendering is broken.

Reply:
Fixed